pages/api/suggestions.js

```diff
@@ -0,0 +1,15 @@
+import suggestionsData from '../../server/suggestions-data.json'
+
+export default function handler(req, res) {
+  const { count } = req.query;
+  if (count) {
+    const slicedSuggestionsData = suggestionsData.slice(0, count);
+    if (slicedSuggestionsData) {
```

Comment:
This will always be true. It's just an array. Maybe check if `count > 0` and remove this check.